### PR TITLE
add substations to elasticsearch indexation

### PR DESCRIPTION
**Pull request by achour94**
**Please check if the PR fulfills these requirements**

- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem?**

**What kind of change does this PR introduce?**

**What is the current behavior?**

**What is the new behavior (if this is a feature change)?**

**Does this PR introduce a breaking change or deprecate an API?**

- [ ] The *Breaking Change* or *Deprecated* label has been added
- [ ] The migration guide has been updated in the github wiki *(What changes might users need to make in their application due to this PR?)*

**Other information**:

### CODE REVIEW

**Review by FranckLecuyer — APPROVED** [review]

**Comment by SlimaneAmar** [comment]
Add a test

File: src/test/java/com/powsybl/network/conversion/server/NetworkConversionTest.java [hunk 1]
```diff
@@ -108,6 +106,136 @@ public void setup() {
...
+                .networkUuid(networkUuid)
+                .variantId(VariantManagerConstants.INITIAL_VARIANT_ID)
+                .id("BBE1AA_w3t")
+                .name("BBE1AA_w3t")
+                .type(IdentifiableType.THREE_WINDINGS_TRANSFORMER.name())
+                .voltageLevels(Set.of(VoltageLevelInfos.builder().id("BBE1TR1").name("BBE1TR1").build(), VoltageLevelInfos.builder().id("BBE1TR2").name("BBE1TR2").build(), VoltageLevelInfos.builder().id("BBE1TR3").name("BBE1TR3").build()))
+                .substations(Set.of(SubstationInfos.builder().id("BBE1AA").name("BBE1AA").build()))
+                .build();
+        assertEquals(expectedEquipmentInfos, equipmentInfos);
+    }
```
Comment (FranckLecuyer) [inline src/test/java/com/powsybl/network/conversion/server/NetworkConversionTest.java:237]: Add also a test for a line present in testCase.xiidm (for example 'BBE1AA1  BBE2AA1  1')

**Review by FranckLecuyer — CHANGES_REQUESTED** [review]

File: src/test/java/com/powsybl/network/conversion/server/NetworkConversionTest.java (hunk 1 above)
Comment (achour94) [reply src/test/java/com/powsybl/network/conversion/server/NetworkConversionTest.java:250]: done

File: src/main/java/com/powsybl/network/conversion/server/NetworkConversionService.java [hunk 2]
```diff
@@ -127,14 +127,15 @@ public NetworkConversionService(@Value("${powsybl.services.case-server.base-uri:
         objectMapper.setInjectableValues(new InjectableValues.Std().addValue(ReporterModelDeserializer.DICTIONARY_VALUE_ID, null));
     }
 
-    private static EquipmentInfos toEquipmentInfos(Identifiable<?> i, UUID networkUuid, String variantId) {
+    public static EquipmentInfos toEquipmentInfos(Identifiable<?> i, UUID networkUuid, String variantId) {
```
Comment (SlimaneAmar) [inline src/main/java/com/powsybl/network/conversion/server/NetworkConversionService.java:130]: Remove `public`

File: src/test/java/com/powsybl/network/conversion/server/NetworkConversionTest.java [hunk 3]
```diff
@@ -108,6 +106,149 @@ public void setup() {
         networkConversionService.setReportServerRest(reportServerRest);
     }
 
+    @Test
+    public void testToEquipmentInfos() {
```
Comment (SlimaneAmar) [inline src/test/java/com/powsybl/network/conversion/server/NetworkConversionTest.java:110]: `NetworkConversionTest` is the class for integration tests.
Move this unit test in `EquipmentInfosServiceTests` class

File: src/test/java/com/powsybl/network/conversion/server/EquipmentInfosServiceTests.java [hunk 4]
```diff
@@ -61,12 +63,159 @@ public void testAddDeleteEquipmentInfos() {
...
+        List<EquipmentInfos> allList = StreamSupport.stream(all.spliterator(), false)
+                .collect(Collectors.toList());
+        assertEquals(infos, allList);
 
         equipmentInfosService.deleteAllOnInitialVariant(NETWORK_UUID);
         assertEquals(0, Iterables.size(equipmentInfosService.findAll(NETWORK_UUID)));
     }
 
+    @Test
+    public void testToEquipmentInfos() {
```
Comment (SlimaneAmar) [inline src/test/java/com/powsybl/network/conversion/server/EquipmentInfosServiceTests.java:77]: Rename by `testEquipmentInfos` ?

**Review by SlimaneAmar — APPROVED** [review]

**Pull request merged**